client/components/PubEdge/PubEdge.js

```diff
@@ -109,41 +109,52 @@ const PubEdge = (props) => {
 		onKeyDown: handleClick,
 		role: 'link',
 		tabIndex: '0',
+		alt: title,
```

Comment:
Most non-`img` tags do not accept an `alt`, and I don't think one is necessarily needed here either, since I believe VoiceOver does a pretty good job of extracting spoken text from these cards.